doc/pmon/sonic_thermal_control_test_plan.md

```diff
@@ -0,0 +1,250 @@
+# Thermal Control Test Plan
+
+## Table of Content
+
+- [Overview](#overview)
+- [Scope](#scope)
+- [Test Structure](#test-structure)
+  - [Setup Configuration](#setup-configuration)
+  - [Ansible and Pytest](#ansible-and-pytest)
+- [Test Cases](#test-cases)
+  - [Show FAN Status Test](#show-fan-status-test)
+  - [Show Thermal Status Test](#show-thermal-status-test)
+  - [FAN Test](#fan-test)
+  - [PSU Absence Test](#psu-absence-test)
+  - [Invalid Policy Format Load Test](#invalid-policy-format-load-test)
+  - [Invalid Policy Value Load Test](#invalid-policy-value-load-test)
+
+
+## Overview
+
+The purpose is to test functionality of thermal control feature on the SONiC switch DUT. The thermal control feature contains 3 major functions: FAN status monitor, thermal status monitor and thermal control policy management.
+
+- FAN status monitor read FAN status via platform API every 60 seconds and save it to redis database. User can fetch FAN status via command line `show platform fanstatus`.
```

Comment:
- suggest to put the function in ' ' for example: 'FAN status monitor'. same for the other functions.
- add s to read

Same comments for the other functions described below

Reply:
the 60 seconds is based on the design, right? is it a configurable value? if not, have we considered it? if yes, should you read it and not use it as hard coded?

Reply:
It's based on design. User cannot configure it for now. Yes, we can do something to allow user configure it, either by policy json files or command line. What is your suggestion? I checked psud and xcvrd, they don't support configure poll interval for now.